static/app/views/settings/settingsIndex.tsx

```diff
@@ -72,10 +71,9 @@ class SettingsIndex extends React.Component<Props> {
     const organizationSettingsUrl =
       (organization && `/settings/${organization.slug}/`) || '';
 
-    const supportLinkProps = {
-      isSelfHosted,
+    const supportLinkProps: SupportLinkExternalProps = {
+      isSelfHosted: true,
       href: LINKS.FORUM,
-      to: `${organizationSettingsUrl}support`,
```

Comment:
I think we were never supposed to pass both href and to to the component, it's just that the types allowed it.

Reply:
Aren't both being passed so that SupportLink can use `href` -> `ExternalLink` and `to` for `HomeLink`? Does this still work when removing it?

Reply:
Ah, nice catch! I missed that.

Reply:
Actually, depending on the type, you would expect just one. I added a ternary to change the different props based on isSelfHosted

Reply:
yeah okay, the ternary works.